indexer/geometry_serialization.hpp

```diff
@@ -153,9 +153,20 @@ namespace serial
   }
 
   inline void const * LoadInnerTriangles(void const * pBeg, size_t count,
-                                         CodingParams const & params, OutPointsT & points)
+                                         CodingParams const & params, OutPointsT & triangles)
   {
-    return LoadInner(&geo_coding::DecodeTriangleStrip, pBeg, count, params, points);
+    triangles.clear();
+    OutPointsT points;
+    void const * res = LoadInner(&geo_coding::DecodeTriangleStrip, pBeg, count, params, points);
+
+    triangles.reserve((count - 2) * 3);
```

Comment:
Please don't reserve is count is less than 2 (the rest of the code is all right).

Reply:
it should not be less than 2, I added 
```C++
CHECK_GREATER_OR_EQUAL(count, 2, ());
```